ffac-autoupdater-wifi-fallback/luasrc/usr/sbin/autoupdater-wifi-fallback

```diff
@@ -136,28 +134,26 @@ if (force or preflight_check()) and not connectivity_check() then
 
   if force or tonumber(unreachable_since) + offset < os.time() then
     autil.log('out', 'going to fallback mode')
-		os.execute('wifi down')
-		os.execute('/etc/init.d/wpad status >/dev/null && /etc/init.d/wpad stop')
+    os.execute('wifi down')
+    os.execute('/etc/init.d/wpad status >/dev/null && /etc/init.d/wpad stop')
     for radio, netlist in pairs(autil.get_available_wifi_networks()) do
-			local radio_config = uci:get_all('wireless', radio)
-			local phy = wireless.find_phy(radio_config)
-			autil.log('out', 'using ' .. phy .. ' to create fallback interface')
-			os.execute('iw phy ' .. phy .. ' interface add fallback type managed')
-			os.execute('ip link set dev fallback up')
+      local radio_config = uci:get_all('wireless', radio)
+      local phy = wireless.find_phy(radio_config)
+      autil.log('out', 'using ' .. phy .. ' to create fallback_if interface')
+      os.execute('iw phy ' .. phy .. ' interface add fallback_if type managed')
+      os.execute('ip link set dev fallback_if up')
       for _, net in ipairs(netlist) do
         switch_to_fallback_mode(radio, net.ssid, net.bssid)
         if run_autoupdater() == 0 then
           break
         end
-				ubus:call('network.interface.fallback', 'down', {})
```

Comment:
are you really sure about not needing the "down" lines? isn't it more clean?

Reply:
It is more clean, but `remove` does trigger a down as well, but we can be explicit here as well.